src/input_formatter/expression_formatter.ts

```diff
@@ -0,0 +1,147 @@
+import { addIssue } from "../issueHandling/issueMapping";
+const allowedExpressionToken = /(bracketLeft|bracketRight|equivalence|implicationRight|implicationLeft|negation|conjunction|disjunction|equal|addition|subtraction|division|multiplication)+/;
+const logicConnector = /(conjunction|disjunction|equivalence|implicationRight|implicationLeft)+/;
+const nonLogicToken = /(bracketLeft|bracketRight|negation)+/;
+
+/**
+	* ---------CurrentExpressionFormatter------------
+	* @param expression written by a user
+	* @return a formatted expression as String
+	*/
+export function expressionFormatter(expression: string) {
+	const preFormattedExpression: string[] = preFormatExpressionFromImput(expression);
+	errorDetector(preFormattedExpression);
+	// TO DO SOME STUFF WITH THE TOKENS AND THE ARRAY; FORMAT THE ARRAY AND CATCH ERRORS
+	const finalFormattedExpression: string[] =
+		replaceExpressionElementsIntoPrologCode(preFormattedExpression);
+	return finalFormattedExpression;
+}
+
+/**
+ * 
+ * @param expression as a input string typed by user
```

Comment:
Even though @param gets generated automatically for the comment, there is no need for it, since one
can just read the method body to see the parameters.

Reply:
Well, if the parameter-documentation includes a helpful description of what the parameter is for, it's legitimate.